core/src/main/java/com/scalar/db/transaction/consensuscommit/ConsensusCommitAdmin.java

```diff
@@ -240,6 +243,58 @@ public void upgrade(Map<String, String> options) throws ExecutionException {
     admin.upgrade(options);
   }
 
+  @Override
+  public void createUser(String username, @Nullable String password, UserOption... userOption) {
+    throw new UnsupportedOperationException("Not supported in the community edition");
```

Comment:
I had the same thought, but I didn't do that because I was not sure for which methods we should add a default implementation. In other words, should we add default implementations for the existing methods (`createTable()`, `getTableMetadata()`, etc.) as well? What do you think?

Reply:
I think only the methods of `AuthAdmin` should have default methods to throw the exception since those features aren't supported in the community edition. As for the existing methods coming from `DistributedTransactionAdmin`, we should keep them available in the community edition and don't need default impl for them. But I might be missing something on the background of this auth feature.

Reply:
Okay, I think that's reasonable. Let me add default implementations to the methods in `AuthAdmin`. Thanks.

Reply:
Fixed in ca46a9d94507f4ba568ad8947f687f1c1f1489b7. Thanks.